src/module_PI/Raspberry_PI/main/PiApp.java

```diff
@@ -1,38 +1,27 @@
 package module_PI.Raspberry_PI.main;
 
+import java.util.*;
 
 public class PiApp {
+	public static Properties GMAIL_CREDENTIALS = new Properties();
 
 	/**
-	 * @param args
+	 * @param args Gmail Username and Password for sending mail
 	 */
 	public static void main(String[] args) {
+		parseArgs(args);
 		SerialInputListener inputListener = new SerialInputListener();
 		Thread listener = new Thread(inputListener, "Listener");
 		listener.start();		
 		SerialOutput output = new SerialOutput();
 		
 		pause(10000);
-		output.sendTestComponentsPacket();
-		output.sendCalibratePacket();
 		output.sendArmPacket();
-		output.sendTriggerPacket();
-		output.sendSilencePacket();
-		
-		while(true){
-			output.sendArmPacket();
-			pause(5000);
-			output.sendTriggerPacket();
-			pause(2000);
-			output.sendSilencePacket();
-			pause(2000);
-			output.sendDisarmPacket();
-			pause(5000);
-			output.sendResetCalibrationPacket();
-			pause(5000);
-			output.sendCalibratePacket();
-			pause(5000);
-		}
+	}
+	
+	private static void parseArgs(String[] args){
+		GMAIL_CREDENTIALS.put("gmailUsername" , args[0]);
```

Comment:
Sorry I didn't catch this before - this method now has the possibility of throwing an exception if the person running the app doesn't provide the username and password. In a previous commit you had a check `if(args.length != numOfArguments) throw new IllegalArgumentException("Syntax is java PiApp <username> <password>");` to catch this issues. Either this check or something similar should probably be put back in.